rent_a_car.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+function calculateMe(form) {
+    const vacation = form.vacationDays.value;
+
+    if ( vacation < 0 || isNaN(vacation) ) {
+        alert('!!! Wrong data !!!');
+        return;
+    }
+
+    const calculation = myPayment(vacation);
```

Comment:
const computedPrice

Reply:
I don't agree with this. But is you insist...
DONE